addons/hr_recruitment_survey/security/hr_recruitment_survey_security.xml

```diff
@@ -1,16 +1,137 @@
+            <field name="name">Survey user input: recruitment officer: unrestricted or in restricted users</field>
+            <field name="model_id" ref="survey.model_survey_user_input"/>
+            <field name="domain_force">[
+                '&amp;', ('survey_id.survey_type', '=', 'recruitment'),
+                '|',  ('survey_id.restrict_user_ids', 'in', user.id),
+                        ('survey_id.restrict_user_ids', '=', False)]</field>
+            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_user'))]"/>
+            <field name="perm_unlink" eval="0"/>
+            <field name="perm_write" eval="0"/>
+            <field name="perm_read" eval="1"/>
+            <field name="perm_create" eval="0"/>
+        </record>
+        <record id="survey_user_input_line_rule_recruitment_user" model="ir.rule">
+            <field name="name">Survey user input line: recruitment officer: unrestricted or in restricted users</field>
+            <field name="model_id" ref="survey.model_survey_user_input_line"/>
+            <field name="domain_force">[
+                '&amp;', ('survey_id.survey_type', '=', 'recruitment'),
+                '|',  ('survey_id.restrict_user_ids', 'in', user.id),
+                        ('survey_id.restrict_user_ids', '=', False)]</field>
+            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_user'))]"/>
+            <field name="perm_unlink" eval="0"/>
+            <field name="perm_write" eval="0"/>
+            <field name="perm_read" eval="1"/>
+            <field name="perm_create" eval="0"/>
+        </record>
+        <record id="survey_invite_recruitment_user" model="ir.rule">
+            <field name="name">Survey invite: recruitment officer: unrestricted or in restricted users</field>
+            <field name="model_id" ref="survey.model_survey_invite"/>
+            <field name="domain_force">[
+                '&amp;', ('survey_id.survey_type', '=', 'recruitment'),
+                '|',  ('survey_id.restrict_user_ids', 'in', user.id),
+                        ('survey_id.restrict_user_ids', '=', False)]</field>
+            <field name="groups" eval="[(4, ref('hr_recruitment.group_hr_recruitment_user'))]"/>
+            <field name="perm_unlink" eval="0"/>
+            <field name="perm_write" eval="1"/>
+            <field name="perm_read" eval="1"/>
+            <field name="perm_create" eval="1"/>
+        </record>
+
+        <!--special rights for recruitment interviewers on recruitment surveys-->
+        <record id="survey_user_input_line_rule_recruitment_interviewer" model="ir.rule">
+            <field name="name">Survey user input line: recruitment interviewer: read survey answers for which they are set as interviewer</field>
+            <field name="model_id" ref="survey.model_survey_user_input_line"/>
+            <field name="domain_force">[
+                ('user_input_id.applicant_id', '!=', False),
+                '|',
```

Comment:
I think this is implied by the next part. I.e., the `applicant_id` cannot be false if it has an `interviewer_ids` or a `job_id` field defined, can it?

```suggestion
```

Reply:
@flch-odoo so the same apply to next rule on `survey_user_input`?

```
<field name="domain_force">[
                ('applicant_id', '!=', False),
                '|',
                    ('applicant_id.interviewer_ids', 'in', user.id),
                    ('applicant_id.job_id.interviewer_ids', 'in', user.id),
                ]</field>
```